logic/src/commonMain/kotlin/com/wire/kalium/logic/data/message/MessageContent.kt

```diff
@@ -20,7 +20,11 @@ sealed class MessageContent {
         val qualifiedConversationId: QualifiedConversationId?
     ) : Client()
 
-    data class Unknown(val encodedData: ByteArray? = null) : Client()
+    data class Unknown( // messages that aren't yet handled properly but stored in db in case
+        val typeName: String? = null,
+        val encodedData: ByteArray? = null
+    ) : Client()
+    object Ignored : Client() // messages that aren't processed in any way
```

Comment:
I don't think receipts (`Read` and `Delivered`), `DataTransfer`, `Reaction` (❤️ likes), and `Availability` should not be in the same group as the `MessageContent.Client` ones.

They are signaling messages. They add content and info to other messages, or to the user sending them, etc.


It's also a bit annoying when trying to get all messages to display:

```
fun getAllMessagesForDisplaying(): List<Message.Client> = TODO("")

getAllMessagesForDisplaying().forEach { message ->
    when(message) {
        is MessageContent.Ignored -> //Why is this here? This shouldn't be here.
    }
}
```

Instead of `object Ignored: Client()`, we could be:

Or, for now, as we don't handle any signaling:

```kotlin
    object Ignored: MessageContent() // instead of Client
```

In the future, we can do like this, without having many breaking changes:

```kotlin
sealed interface Signaling: MessageContent(){ // Instead of Ignored
    // different signaling types
}
```


By doing that, it's easier to handle different and handle these different types.

We can handle:

```kotlin
fun handleSignalingMessage(signalingMessage: Signaling){
    TODO("Handle signaling message")
    // receipts, availability, etc.
}
```